pkg/util/sys/device.go

```diff
@@ -91,6 +102,59 @@ func ListDevices(executor exec.Executor) ([]string, error) {
 	return strings.Split(devices, "\n"), nil
 }
 
+func GetCephVolumeInventory(executor exec.Executor) (*[]CephVolumeDeviceData, error) {
+	cmd := "ceph-volume inventory --format plain"
+	inventory, err := executor.ExecuteCommandWithOutput(false, cmd, "ceph-volume", "inventory", "--format", "json")
+	if err != nil {
+		return nil, fmt.Errorf("failed to execute ceph-volume inventory: %+v", err)
+	}
+
+	// Slice to store device properties
+	CVStorageDevices := []CephVolumeDeviceData{}
+
+	// Get a slice to store the json data
+	bInventory := []byte(inventory)
+	var f interface{}
+	err = json.Unmarshal(bInventory, &f)
```

Comment:
Do we even need to unmarshal the c-v output? What if we just take the raw output and save it to the configmap? But if we do need to deserialize it, I hope adding the json properties above will greatly simplify the deserialization.

Reply:
See my comments in you previous suggestion.